Source/DafnyCore/Generic/Reporting.cs

```diff
@@ -218,7 +218,8 @@ public class ConsoleErrorReporter : BatchErrorReporter {
           errorLine += $" {msg} {tok.TokenToString(Options)}";
         }
 
-        if (Options.Verbose && false) { // Need to control tests better before we enable this
+        if (Options.Verbose) { // Need to control tests better before we enable this
+          errorLine += "\nID: " + errorId;
```

Comment:
I think it's a bit confusing to have just "ID:" on a separate line - less obvious that's it's an error id about the previous line.  I think this would be clearer:

```
formatting1-3.dfy(1,0): Warning: File contains no code (ID: ...)
```

This would probably be even better, to put the id next to "Warning" or "Error", but may require more plumbing:

```
formatting1-3.dfy(1,0): Warning (ID: ...): File contains no code 
```

Reply:
I'll put it after the message, so the important part of the message is upfront.
It is still only included when --verbose is on, so it won't affect existsing tests, but will take a bit more sedding to tease it out of the message.